docs(cells): Fixed paragraph describing standard cell representation (#3442)
tact-lang/tact: docs/src/content/docs/book/cells.mdx

Then, the data bits of the cell themselves are serialized as $\lceil\frac{b}{8}\rceil$ 8-bit octets (bytes). If $b$ is not a multiple of eight, a binary `1` followed by up to six binary `0`s is appended to the data bits.

> Next, for every referenced cell, two bytes in Big Endian format store the depth of the refs, i.e., the number of cells between the root of the cell tree (the current cell) and the deepest reference, including it. For example, a cell containing only one reference and no further references would have a depth of 1, while the referenced cell would have a depth of 0.

Reviewer: ```suggestion
Next, for every referenced cell, 2 bytes in Big Endian format store the depth of the refs, i.e., the number of cells between the root of the cell tree (the current cell) and the deepest reference, including it. For example, a cell containing only one reference and no further references would have a depth of 1, while the referenced cell would have a depth of 0.
```

> Finally, for every referenced cell, the [SHA-256][sha-2] hash of its standard representation is stored, occupying 32 bytes per referenced cell, recursively repeating the said algorithm. Note that cyclic cell references are not allowed, so this recursion always terminates in a well-defined manner.
> 
> If there are no referenced cells, no depth or hashes are stored.
> 
> :::note
> 
> This process might differ for exotic cells, or cells, that contain exotic cells in their subtree. [Kinds of cells](#cells-kinds)
> 
> :::

Reviewer: ```suggestion
Finally, for every referenced cell, the [SHA-256][sha-2] hash of its standard representation is stored, occupying 32 bytes per referenced cell, recursively repeating the said algorithm. Note that cyclic cell references are not allowed, so this recursion always terminates in a well-defined manner. If there are no referenced cells, neither depths nor hashes are stored.

:::note

  This process of obtaining the standard cell representation differs for cells that contain [exotic cells](#cells-kinds) in their subtree or are exotic themselves.

:::
```

Then, the data bits of the cell themselves are serialized as $\lceil\frac{b}{8}\rceil$ 8-bit octets (bytes). If $b$ is not a multiple of eight, a binary `1` followed by up to six binary `0`s is appended to the data bits.

Next, for every referenced cell, 2 bytes in Big Endian format store the depth of the refs, i.e., the number of cells between the root of the cell tree (the current cell) and the deepest reference, including it. For example, a cell containing only one reference and no further references would have a depth of 1, while the referenced cell would have a depth of 0.

> Finally, for every referenced cell, the [SHA-256][sha-2] hash of its standard representation is stored, occupying 32 bytes per referenced cell, recursively repeating the said algorithm. Note that cyclic cell references are not allowed, so this recursion always terminates in a well-defined manner. If there are no referenced cells, neither depths nor hashes are stored.

Reviewer 2: ```suggestion
Finally, for every referenced cell, the [SHA-256][sha-2] hash of its standard representation is stored, occupying 32 bytes per referenced cell, recursively repeating the said algorithm. Note that cyclic cell references are not allowed, so this algorithm always terminates. If there are no referenced cells, neither depths nor hashes are stored.
```
